netprofile/netprofile/common/subscribers.py

```diff
@@ -73,7 +74,13 @@ def on_response(event):
             img_src = ["'self'"]
             frame_src = ["'self'"]
 
-            if rt_host:
+            if rt_url:
```

Comment:
This is wrong. Proper logic here is:
1. If netprofile.rt.url is supplied, assign it to rt_url verbatim.
2. If not, make rt_url as before.